presto-kafka/src/main/java/io/prestosql/plugin/kafka/schema/MapBasedTopicSchemaProvider.java

```diff
@@ -23,12 +24,12 @@
 
 import static java.util.Objects.requireNonNull;
 
-public class MapBasedTableDescriptionSupplier
-        implements TableDescriptionSupplier
+public class MapBasedTopicSchemaProvider
+        implements TopicSchemaProvider
```

Comment:
Extract rename to separate commit.

Reply:
Now the name is unchanged but they were moved to a different package, should that be extracted to a separate commit?